contracts/staking/ExampleRewardCalculator.sol

```diff
@@ -0,0 +1,31 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.25;
+
+import {IRewardCalculator} from "./interfaces/IRewardCalculator.sol";
+
+contract ExampleRewardCalculator is IRewardCalculator {
+    uint256 public constant SECONDS_IN_YEAR = 31536000;
```

Comment:
I assume we don't need to account for leap years in this initial staking reward curve.

Reply:
I think it would add a whole lot of complexity without any real tangible benefit